content/en/docs/instrumentation/go/manual.md

```diff
@@ -720,6 +720,48 @@ func init() {
 }
 ```
 
+### Registering Views
+
+A View provides SDK users with the flexibility to customize the metrics that are
+output by the SDK. They can customize which metric instruments are to be
+processed or ignored. They can customize aggregation and what attributes are to
+be reported on metrics.
```

Comment:
This is copy pasted from: https://opentelemetry.io/docs/concepts/signals/metrics/#views